pwndbg/commands/dev.py

```diff
@@ -65,3 +66,20 @@ def dev_dump_instruction(address=None, force_emulate=False, no_emulate=False) ->
         if instructions:
             insn = instructions[0]
             print(repr(insn))
+
+
+parser = argparse.ArgumentParser(description="Set the log level.")
+parser.add_argument(
+    "level",
+    type=str,
+    nargs="?",
+    choices=["debug", "info", "warning", "error", "critical"],
+    default="warning",
+    help="The log level to set.",
+)
+
+
+@pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.DEV)
+def log_level(level: str) -> None:
```

Comment:
Should this be a command or a config variable?

Reply:
A command is the most straight forward way to do this right now. We need to call the `setLevel` method of the root logger to change the log level, while config parameters don't really call any functions.